src/app_task.cpp

```diff
@@ -356,3 +386,67 @@ void AppTask::DispatchEvent(const AppEvent &event)
 		LOG_INF("Event received with no handler. Dropping event.");
 	}
 }
+
+void AppTask::UpdateTemperatureClusterState(const AppEvent &)
+{
+	static constexpr int16_t kTemperatureMeasurementAttributeMaxValue = 0x7fff;
```

Comment:
Where do these come from? The current spec provides a bit different numbers. Also, it may be clearer to represent a negative value as a signed literal.

Reply:
I think if you notice any spec non-compliant code in existing samples, it's probably better to do it properly at a webinar and fix the samples later than follow the samples. This is to avoid difficult questions during the webinar :)